sdk/tables/azure-data-tables/inc/azure/data/tables/table_client.hpp

```diff
@@ -134,13 +134,13 @@ namespace Azure { namespace Data { namespace Tables {
      * @brief Upsert specified entity in a table.
      *
      * @param tableEntity The TableEntity to upsert.
-     * @param options Optional parameters to execute this function.
+     * @param options Upsert kind(update/merge).
```

Comment:
I'd add a comment that default is Update

Reply:
done

Reply:
If there's a consensus, does this default match other languages?

Just looking at .NET, the default seems to be Merge there, not Update:
https://github.com/Azure/azure-sdk-for-net/blob/1fbfe17e186f87e5422653b877c2aef178adf159/sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.net8.0.cs#L55-L58

Reply:
FWIW, GoLang and Java also both default to `Merge` instead of `Update`, when the option isn't set/passed in:
https://github.com/Azure/azure-sdk-for-go/blob/ea1a0e789aff76169459041fec0a0732b534f9c0/sdk/data/aztables/client.go#L425-L429

https://github.com/Azure/azure-sdk-for-java/blob/f6a6562b01811506f142018971094b1ab3983fb3/sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClient.java#L837-L846